tests/testthat/test-independent_test_gs_design_wlr.R

```diff
@@ -0,0 +1,159 @@
+    lrk$bounds %>% dplyr::mutate_if(is.numeric, round, digits = 2) %>% dplyr::select(Analysis, Bound, Probability) %>%
+    tidyr::pivot_wider(names_from = Bound, values_from = Probability)
+  # $bounds %>%
+  #   dplyr::mutate_if(is.numeric, round, digits = 2) %>%
+  #   # dplyr::select(Analysis, Bound, Time, N, Events, AHR, Probability) %>%
+  #   dplyr::select(Analysis, Bound, N, Events, AHR, Probability) %>%
+  #   tidyr::pivot_wider(names_from = Bound, values_from = Probability)
+  #FH(0,1)
+  fh01 <- gsDesign2::gs_design_wlr(
+    enrollRates = enrollRates,
+    failRates = failRates,
+    weight = function(x, arm0, arm1) {
+      gsDesign2::wlr_weight_fh(x, arm0, arm1, rho = 0, gamma = 1)
+    },
+    ratio = ratio, alpha = alpha, beta = beta,
+    upar = x$upper$bound,
+    lpar = x$lower$bound,
+    analysisTimes = analysisTimes
+  )
+  fh01_bnd <-
+    fh01$bounds %>% dplyr::mutate_if(is.numeric, round, digits = 2) %>% dplyr::select(Analysis, Bound, Probability) %>%
+    tidyr::pivot_wider(names_from = Bound, values_from = Probability)
+  
+  # $bounds %>%
+  #   dplyr::mutate_if(is.numeric, round, digits = 2) %>%
+  #   select(Analysis, Bound, Time, N, Events, AHR, Probability) %>%
+  #   tidyr::pivot_wider(names_from = Bound, values_from = Probability)
+  #FH(0,0.5)
+  fh0d5 <- gsDesign2::gs_design_wlr(
+    enrollRates = enrollRates,
+    failRates = failRates,
+    weight = function(x, arm0, arm1) {
+      gsDesign2::wlr_weight_fh(x, arm0, arm1, rho = 0, gamma = 0.5)
+    },
+    ratio = ratio, alpha = alpha, beta = beta,
+    upar = x$upper$bound,
+    lpar = x$lower$bound,
+    analysisTimes = analysisTimes
+  )
+  fh0d5_bnd <- 
+    fh0d5$bounds %>% dplyr::mutate_if(is.numeric, round, digits = 2) %>% dplyr::select(Analysis, Bound, Probability) %>%
+    tidyr::pivot_wider(names_from = Bound, values_from = Probability)
+  # $bounds %>%
+  #   dplyr::mutate_if(is.numeric, round, digits = 2) %>%
+  #   select(Analysis, Bound, Time, N, Events, AHR, Probability) %>%
+  #   tidyr::pivot_wider(names_from = Bound, values_from = Probability)
+  #FH(0.5,0.5)
+  fh5d5 <- gsDesign2::gs_design_wlr(
+    enrollRates = enrollRates,
+    failRates = failRates,
+    weight = function(x, arm0, arm1) {
+      gsDesign2::wlr_weight_fh(x, arm0, arm1, rho = 0.5, gamma = 0.5)
+    },
+    ratio = ratio, alpha = alpha, beta = beta,
+    upar = x$upper$bound,
+    lpar = x$lower$bound,
+    analysisTimes = analysisTimes
+  )
+  fh5d5_bnd <-
+    fh5d5$bounds %>% dplyr::mutate_if(is.numeric, round, digits = 2) %>% dplyr::select(Analysis, Bound, Probability) %>%
+    tidyr::pivot_wider(names_from = Bound, values_from = Probability)
+  # $bounds %>%
+  #   dplyr::mutate_if(is.numeric, round, digits = 2) %>%
+  #   select(Analysis, Bound, Time, N, Events, AHR, Probability) %>%
+  #   tidyr::pivot_wider(names_from = Bound, values_from = Probability)
+
+  #logrank part
+  # expect_equal(object = as.numeric(lrk$N), expected = rep(386, 3), tolerance = 3)

```

Comment:
Hi Hongtao, can we remove these comments? Since they are no longer needed and GitHub has version history.